charts/cortx/values.yaml

```diff
@@ -84,6 +84,29 @@ externalKafka:
   adminUser: admin
   adminSecretName: kafka_admin_secret
 
+# CORTX Hare component configuration
+hare:
+  hax:
+    ports:
+      http:
+        # -- The HTTP scheme used to configure the Hax HTTP endpoint. Valid values are `http` or `https`.
+        scheme: https
+        # -- The port number of the Hax HTTP endpoint.
+        port: 22003
+    resources:
+      # -- Configure the resource limits for Hax containers. This applies to all Pods that run Hax containers.
+      ## Example:
+      ## limits:
+      ##   cpu: 100m
+      ##   memory: 128Mi
+      limits: {}
```

Comment:
I don't think we want these values to be an empty object here. The rest of the component resources blocks are not empty objects in this values.yaml file and it's confusing if hare is different than the rest of them as a one-off.

Based on the [discussion around requests & limits](https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/1053753641/CORTX+Container+Resource+Limits) _([previous](https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/824476019/CORTX+Container+Resource+limits+-+Obsolete))_,  I think we want to keep the suggested values in values.yaml to guarantee a minimal baseline (that can be overridden with higher defaults or `{}` at deploy time).